ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Query.hs

```diff
@@ -162,16 +231,24 @@ queryDecodeNodeToClient codecConfig queryVersion blockVersion
         blockVersion
       return (SomeSecond (BlockQuery blockQuery))
 
-instance SerialiseResult blk (BlockQuery blk) => SerialiseResult blk (Query blk) where
+instance (SerialiseResult blk (BlockQuery blk), Serialise (HeaderHash blk), Typeable blk) => SerialiseResult blk (Query blk) where
```

Comment:
Would it make sense to [separate this into several lines](https://github.com/input-output-hk/ouroboros-network/blob/master/ouroboros-consensus/docs/StyleGuide.md#L430)?

Reply:
Ugh, links to lines in Markdown files do not seem to work. I was referring to this

![image](https://user-images.githubusercontent.com/175315/133235793-5f385213-c86a-42d0-96b8-e1068442ccde.png)

Reply:
Done!